### PR TITLE
Move release webhook to notification

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

**Comment by 6543** [comment]
```
mode, _ := models.AccessLevel(doer, rel.Repo)
	if err := webhook.PrepareWebhooks(rel.Repo, models.HookEventRelease, &api.ReleasePayload{
		Action:     api.HookReleaseUpdated,
		Release:    rel.APIFormat(),
		Repository: rel.Repo.APIFormat(mode),
		Sender:     doer.APIFormat(),
	}); err != nil {
		log.Error("PrepareWebhooks: %v", err)
	}
```
exists 3 times only differ in `Action: xy`

does it make sence to move this in a seperate function wich is used by all three funcitons?

**Comment by lunny** [comment]
@6543 done.

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
CI.restart()

**Comment by techknowlogick** [comment]
restarted

File: modules/notification/webhook/webhook.go [hunk 1]
```diff
@@ -10,7 +10,6 @@ import (
 	"code.gitea.io/gitea/modules/notification/base"
 	"code.gitea.io/gitea/modules/setting"
 	api "code.gitea.io/gitea/modules/structs"
-	"code.gitea.io/gitea/modules/webhook"
```
Comment (6543) [inline modules/notification/webhook/webhook.go:14]: @lunny it works on your current branch but not after merge!
-> we still need this import

**Review by 6543 — CHANGES_REQUESTED** [review]

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
@techknowlogick can you reviwe&merge?

**Review by zeripath — APPROVED** [review]

**Pull request merged**